com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md

```diff
@@ -16,7 +16,14 @@ The Render Pipeline Debugger window.
 
 ## Using the Render Pipeline Debugger window
 
-To open the Render Pipeline Debugger window in the Editor, go to  **Window > Analysis > Rendering Debugger**. You can also open this window at runtime in Play Mode, or in the standalone Unity Player on any device on **Development build**. Use the keyboard shortcut Ctrl+Backspace (Ctrl+Delete on macOS) or press L3 and R3 (Left Stick and Right Stick) on a controller to open the window. You can disable the shortcut through the [enableWindowHotkey variable](https://docs.unity3d.com/Packages/com.unity.render-pipelines.core@12.0/api/UnityEngine.Rendering.DebugManager.html#UnityEngine_Rendering_DebugManager_enableWindowHotkey).
+To open the Render Pipeline Debugger window in the Editor, you must first enable **Runtime Debug Shaders** in **HDRP Global Settings**:
+
+1. Go to **Edit > Project Settings > Graphics** 
+2. Click **HDRP Settings** and enable **Runtime Debug Shaders**.
+
+To open the Render Pipeline Debugger window, go to **Window > Analysis > Rendering Debugger**. 
+
+You can also open this window at runtime in Play Mode, or in the standalone Unity Player on any device on **Development build**. Use the keyboard shortcut Ctrl+Backspace (Ctrl+Delete on macOS) or press L3 and R3 (Left Stick and Right Stick) on a controller to open the window. You can disable the shortcut through the [enableWindowHotkey variable](https://docs.unity3d.com/Packages/com.unity.render-pipelines.core@12.0/api/UnityEngine.Rendering.DebugManager.html#UnityEngine_Rendering_DebugManager_enableWindowHotkey).
```

Comment:
I think you should change the URL version.

```suggestion
You can also open this window at runtime in Play Mode, or in the standalone Unity Player on any device on **Development build**. Use the keyboard shortcut Ctrl+Backspace (Ctrl+Delete on macOS) or press L3 and R3 (Left Stick and Right Stick) on a controller to open the window. You can disable the shortcut through the [enableWindowHotkey variable](https://docs.unity3d.com/Packages/com.unity.render-pipelines.core@latest/api/UnityEngine.Rendering.DebugManager.html#UnityEngine_Rendering_DebugManager_enableWindowHotkey).
```

Reply:
I'm actually changing this API reference in my PR so I've fixed it there. https://github.com/Unity-Technologies/Graphics/pull/5339/files#diff-4755bb60d17bf206f39d7348502d2f8dfe25ccfe7b24f46a05879c3222f151e4L19

Reply:
nice :)